docs/rules/no-empty.md

```diff
@@ -65,6 +65,58 @@ try {
 
 Since you must always have at least a `catch` or a `finally` block for any `try`, it is common to have empty statements when execution should continue regardless of error.
 
+### Options
+
+`methods` set to `true` will warn for empty methods. This option is set to `false` by default.
+
+```json
+{
+    "no-empty": [2, { "methods": true } ]
+}
+```
+
+The following patterns are considered problems when `methods` is set to `true`:
+
+```js
+/*eslint no-empty: [2, { methods: true }]*/
+
+var foo = {
+    bar: function() {}    /*error Empty block statement.*/
+}
+
+var foo = {
+    bar() {}              /*error Empty block statement.*/
+}
+
+var foo = {
+    bar: () => {}         /*error Empty block statement.*/
+}
+```
+
+The following patterns are not considered problems when `methods` is set to `true`:
+
+```js
+/*eslint no-empty: [2, { methods: true }]*/
+
+var foo = {
```

Comment:
Updated without changing any of the language regarding specifying ES6 methods in particular - can add that in pending the response to my question above.